packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -254,7 +254,7 @@ const AutocompletePaper = experimentalStyled(Paper, {
   /* Styles applied to the Paper component. */
   ...theme.typography.body1,
   overflow: 'auto',
-  margin: '4px 0',
+  margin: '1px 0',
```

Comment:
Why not remove the margin? from what I check in autocomplete, removing margin does not affect anything.

Reply:
It feels slightly better and give a tip for the developers on how to add a margin. No strong objections on removing it.

Reply:
I suggest to remove it (1px might seem like a bug for pixel perfect people 😂). Better create a demo how to add space in between input and paper instead.

Reply:
the rest looks good to me.